doc/glossary.rst

```diff
@@ -54,6 +54,12 @@ general neuroimaging concepts. If you think a term is missing, please consider
     dipole
         See :term:`equivalent current dipole`.
 
+    dSPM
+        Dynamic statistical parametric mapping (abbr. ``dSPM``) gives a noise-
+        normalized :term:`source estimate` at a given source location. dSPM 
+        can be obtained by applying the inverse operator to the signal 
+        covariance matrix. 
```

Comment:
This is a little bit vague. Do you mean the "dSPM normalization terms can be obtained ..." ?